samples/client/petstore/elm-0.18/src/Data/Order_.elm

```diff
@@ -65,20 +63,21 @@ orderEncoder model =
 statusDecoder : Decoder Status
 statusDecoder =
     Decode.string
-        |> Decode.andThen (\str ->
-            case str of
-                "placed" ->
-                    Decode.succeed Placed
+        |> Decode.andThen
```

Comment:
This I will update in the templates as well (no need to that in this PR as more templates will change)